test/e2e/policyrecommendation_test.go

```diff
@@ -172,51 +171,51 @@ func TestPolicyRecommendation(t *testing.T) {
 
 // Example output: Successfully created policy recommendation job with ID e998433e-accb-4888-9fc8-06563f073e86
```

Comment:
`ID` -> `name`
Names should have prefix `pr-`?

Reply:
Fixed.